Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -561,14 +575,15 @@ def main():
 
     params = demisto.params()
     base_url = urljoin(params.get('url').rstrip('/'), '/')
+    timeout = int(demisto.get(demisto.params(), 'timeout'))
```

Comment:
```suggestion
    timeout = arg_to_number(demisto.get(demisto.params(), 'timeout'))
```

lets give a default value here as well.
also I don't see a reason to use `demisto.get` since `params` is not a nested json and the timeout should be first level arg.

Reply:
fixed